### ISSUE TITLE
Name change history does not appear in NameCheck tickets

### ISSUE THREAD

**Issue by rhymu8354** [rhymu8354/OmniaRegna#2]

**Comment by rhymu8354** [comment]
Example:

![image](https://user-images.githubusercontent.com/41497796/71500157-973e7380-2818-11ea-9475-581f39a5aed6.png)

Here, the player's name was originally "rhymu", then changed to "rhymu2", which opened the ticket, and then changed again to "rhymu2b".

Problems:

1. The `was` field should be shown in the `Details` for `username` changes.
2. The `By Whom` shows `GM` rather than `Player1`.
3. It's a bit confusing knowing what the name was before and after each change.

Solutions:

1. When a `NameCheck` ticket is created because of a name change, the history should begin with two changes: `create` and `username`, with the `username` having a `was` holding the old player name.
2. Make sure the `By Whom` is set to the player and not `GM` when the player is the one changing the name.  The `username` change should include a `who` field.
3. For the `Details` fields of `username` changes to a `NameCheck` ticket, display both what the name was previously and what it changed to, using the `was` field of the change and either the `was` field of the next `username` change or the `username` field of the ticket itself.

**Comment by rhymu8354** [comment]
I've moved the code that creates and updates `NameCheck` tickets from the Coordinator to the Executor, and made it so that new `NameCheck` tickets created due to a name change begin with both `create` and `username` entries in the history.

I also fixed the `By Whom` to show the player when the player changes their name.

I still need to update the front-end to show the name changes in the ticket details.
